src/shared/components/dropdown/select/SelectDropdownItem.tsx

```diff
@@ -0,0 +1,111 @@
+'use client';
+
+import { cva } from 'class-variance-authority';
+import { KeyboardEvent, ReactNode } from 'react';
+import useDropdownBaseContext from '@/shared/components/dropdown/hooks/useDropdownBaseContext';
+import useSelectContext from '@/shared/components/dropdown/hooks/useSelectContext';
+import {
+  dropdownItemHoverBase,
+  dropdownItemBase,
+  dropdownItemShadowStyle,
+} from '@/shared/components/dropdown/styles/dropdownItem';
+import { cn } from '@/shared/utils/cn';
+
+export const dropdownItemVariants = cva(dropdownItemBase, {
+  variants: {
+    variants: {
+      basic: 'rounded-12 px-20 py-12 body-14 sm:body-16',
+      shadow: dropdownItemShadowStyle,
+    },
+    disabled: {
+      true: 'cursor-not-allowed bg-gray-25 text-gray-400',
+      false: 'text-gray-800 hover:font-bold',
+    },
+    selected: {
+      true: 'font-bold text-primary-500',
+      false: '',
+    },
+  },
+  compoundVariants: [
+    {
+      disabled: false,
+      className: dropdownItemHoverBase,
+    },
+  ],
+  defaultVariants: {
+    variants: 'basic',
+    disabled: false,
+    selected: false,
+  },
+});
+
+interface SelectDropdownItemProps {
+  children: ReactNode;
+  value: string;
+  disabled?: boolean;
+}
+
+/**
+ * ## SelectDropdownItem
+ *
+ * @description
+ * SelectDropdown 내에서 하나의 선택 가능한 옵션을 표현하는 컴포넌트입니다.
+ *
+ * - `value`를 기준으로 현재 선택 상태를 판단합니다.
+ * - 클릭 또는 키보드(Enter / Space)로 선택할 수 있습니다.
+ * - 선택 시 값을 변경하고 드롭다운을 닫습니다.
+ * - `disabled` 상태인 경우 선택 및 포커스를 차단합니다.
+ *
+ * @param children - 옵션에 표시될 콘텐츠 (텍스트, 아이콘 등)
+ * @param value - 옵션의 실제 선택 값
+ * @param disabled - 옵션 비활성화 여부
+ *
+ * @example
+ * ```tsx
+ * <SelectDropdownItem value={문화 · 예술}>🎨 문화 · 예술</SelectDropdownItem>
+ * ```
+ */
+export default function SelectDropdownItem({
+  children,
+  value,
+  disabled = false,
+}: SelectDropdownItemProps) {
+  const { setIsOpen } = useDropdownBaseContext();
+  const { value: selectedValue, setValue, variants } = useSelectContext();
+
+  const isSelected = selectedValue === value;
+
+  const selectOption = () => {
+    if (disabled) {
+      return;
+    }
+    setValue(value);
+    setIsOpen(false);
+  };
+
+  const handleKeyDown = (e: KeyboardEvent<HTMLLIElement>) => {
+    if (disabled) {
+      return;
+    }
+
+    if (e.key !== 'Enter' && e.key !== ' ') {
+      return;
+    }
+
+    e.preventDefault();
+    selectOption();
+  };
```

Comment:
화살표 키(↑, ↓)도 추가해서 이전/다음 항목으로 포커스 이동하도록 해도 좋을 것 같아요!


```suggestion
  const handleKeyDown = (e: KeyboardEvent<HTMLLIElement>) => {
    if (disabled) {
      return;
    }

    switch (e.key) {
      case 'Enter':
      case ' ':
        e.preventDefault();
        selectOption();
        break;
      case 'ArrowDown':
      case 'ArrowUp':
        // 다음/이전 항목으로 포커스 이동
        e.preventDefault();
        break;
    }
```

Reply:
tab으로 전환돼도, 화살표 키로 전환하는 게 있으면 좋지 않을까요?
tab은 다른 요소 아래로만 이동하니까요.

혹시 `ActionDropdownList`은 아름님이 판단하셔서 추가했을 때 곤란한 문제가 생기면 추가하지 않으셔도 괜찮아요!